src/t8_geometry/t8_geometry_implementations/t8_geometry_linear.hxx

```diff
@@ -52,24 +52,28 @@ struct t8_geometry_linear: public t8_geometry_with_vertices
 
   /**
    * Map a point in the reference space \f$ [0,1]^\mathrm{dim} \to \mathbb{R}^3 \f$.
-   * \param [in]  cmesh      The cmesh in which the point lies.
-   * \param [in]  gtreeid    The global tree (of the cmesh) in which the reference point is.
-   * \param [in]  ref_coords  Array of \a dimension many entries, specifying a point in \f$ [0,1]^\mathrm{dim} \f$.
+   * \param [in]  cmesh       The cmesh in which the point lies.
+   * \param [in]  gtreeid     The global tree (of the cmesh) in which the reference point is.
+   * \param [in]  ref_coords  Array of \a dimension x \a num_coords many entries, specifying points in \f$ [0,1]^\mathrm{dim} \f$.
+   * \param [in]  num_coords  Amount of points of /f$ \mathrm{dim} /f$ to map.
    * \param [out] out_coords  The mapped coordinates in physical space of \a ref_coords.
    */
   virtual void
-  t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords, double out_coords[3]) const;
+  t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords, const size_t num_coords,
+                    double out_coords[3]) const;
```

Comment:
I took the `linear` geometry as example since it is the mostly used right now.

I think the output argument `double out_coords[3]` is misleading for the user. I suggest
```suggestion
                    double *out_coords) const;
```
and update the documentation string above. The compiler probably does not complain
since behind the curtain it is the same.

Reply:
This change must also be updated for the other geometry implementations.

Reply:
done